### PR TITLE
attempt to fix hardcoded icons

### PR DESCRIPTION
**Pull request by bilelmoussaoui**
Few  stuff are still needed before this can be merged!

Some icons are missing under
- 16x16/apps/
- 22x22/apps/
- 32x32/apps/
- 48x48/apps/
- 256x256/apps/
- scalable/apps

In 16, 22, 32, 48, 256 the icons must be named `ibus-hunspell-table.png` : I can create them using the original SVG icons, but they might look blurry? 
In scalable folder, a symbolic icon must be added for Gnome desktop
As i suck as a designer, maybe someone else can provide them? 

@mike-fabian What should i add/modify/remove?

### CODE REVIEW

File: engine/emoji-picker [hunk 1]
```diff
@@ -16,8 +16,8 @@
 #  You should have received a copy of the GNU General Public License
```
Comment (mike-fabian) [inline engine/emoji-picker]: Oh, I made a mistake, this file is generated and shouldn’t have been in git.
I removed it now and added it to .gitignore.

Comment (bil-elmoussaoui) [inline engine/emoji-picker]: Oops i will fix that! What shall we do for the missing icons?

Comment (mike-fabian) [inline engine/emoji-picker]: Creating them from the original SVG icons should be fine for the moment.

I don’t know where the original icon comes from, I didn’t create it, it was already there when
I started  working on ibus-typing-booster. I would like to replace it with something nicer looking,
maybe a rocket? ( https://en.wikipedia.org/wiki/Booster_(rocketry) ) But I have
never designed icons either. So I think for the moment creating .png files from the original .svg
should be OK.

Comment (bil-elmoussaoui) [inline engine/emoji-picker]: @mike-fabian alright i will get in touch with a designer and see if he can do some better icons based on the original design. But for now i will generate the png icons from svg ones and remove the symbolic icon so this PR can be merged

Comment (bil-elmoussaoui) [inline engine/emoji-picker]: @mike-fabian Quick update, the designer agreed to work on this. I suggest to wait until the icons are ready :+1:

Comment (mike-fabian) [inline engine/emoji-picker]: OK, thank you!

**Comment by mike-fabian** [comment]
Any news about the icons?

**Comment by mike-fabian** [comment]
Something similar is included in the 2.0.0 release now.
There is only one icon, ibus-typing-booster.svg now, all the language specific icons
have been removed because there is only one engine now.

The icon has the same style as before and the same icon is also used for emoji-picker.

The ibus-typing-booster.svg and several ibus-typing-booster.png files are installed  here now:

/usr/share/icons/hicolor/32x32/apps/ibus-typing-booster.png
/usr/share/icons/hicolor/256x256/apps/ibus-typing-booster.png
/usr/share/icons/hicolor/64x64/apps/ibus-typing-booster.png
/usr/share/icons/hicolor/48x48/apps/ibus-typing-booster.png
/usr/share/icons/hicolor/scalable/apps/ibus-typing-booster.svg
/usr/share/icons/hicolor/16x16/apps/ibus-typing-booster.png
/usr/share/icons/hicolor/128x128/apps/ibus-typing-booster.png
/usr/share/icons/hicolor/22x22/apps/ibus-typing-booster.png

The .png files have been generated from the .svg using imagemagick like this:

    convert ibus-typing-booster.svg -resize 48x48 -transparent white ibus-typing-booster.png

I don’t really like the  design  of the icon, the keyboard and the globe shown in the icon 
don’t look really nice  to me. And of course it is not so nice that exactly the same
icon is used for emoji-picker.  I think I would prefer  something like a rocket as 
an  icon for ibus-typing-booster,  the rocket  emoji is also used in the  Gnome panel
when ibus-typing-booster is selected.

And of course emoji-picker should have its own icon.

But it is better than before, therefore I close this pull request now.

**Pull request closed without merging**